### ISSUE TITLE
litemount bug

### ISSUE THREAD

**Issue by j3nn14er** [xod-wow/LiteMount#73]
What happens?
When in filter for group, when all boxes are checked it shows all mounts; when no boxes are checked it shows no mounts.

What did you expect to happen instead?
Expect when no boxes are checked, that group is not considered in the filtered results, and when all boxes are checked then it only shows the mounts that are in all of the groups. This makes sense for some other filter options, but seems like having all the boxes checked/unchecked doesn't affect the specific filter but just shows all mounts or no mounts. 

Did it work in a previous version of LiteMount? If so, what was the last version that worked?
Just downloaded today, never tried previous versions

**Comment by mbattersby** [comment]
Do you have only one group?

**Comment by mbattersby** [comment]
I agree it is a little bit confusing and doesn't suit the check all / uncheck all very well.

But for your second part, having boxes checked or unchecked does affect the mounts. Screenshot:

![image](https://user-images.githubusercontent.com/1300207/134831226-3e34536e-a0b0-4e0a-a0af-847152d6dfa4.png)

Is that not what's happening for you?

**Comment by mbattersby** [comment]
I think perhaps the best solution is to add an extra option "NONE" that covers all the mounts that aren't in groups.

**Comment by mbattersby** [comment]
6f7ec84 adds NONE option to group menu. Didn't make 9.1.14, should be in 9.1.15.

**Comment by mbattersby** [comment]
NONE option is in 9.1.15. Closing.
